src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagStoreEntry.cs

```diff
@@ -35,6 +36,13 @@ public CustomTagStoreEntry(int key, string path, string vr, CustomTagLevel level
         /// </summary>
         public string VR { get; set; }
 
+        /// <summary>
+        /// Private Creator of this tag.
+        /// It's identification code of private tag implementer, only apply to private tag.
+        /// Please refer to http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_7.8.html for more details.
```

Comment:
The comment starting at `It's identification code...` should be in a `<remarks>...</remarks>` element